plugins/services/src/js/pages/task-details/TaskLogsContainer.js

```diff
@@ -50,19 +51,36 @@ class TaskLogsContainer extends mixin(StoreMixin) {
       return <Loader />;
     }
 
-    const {props} = this;
+    const {
+      directory,
+      limitLogFiles,
+      params,
+      routes,
+      selectedLogFile,
+      task
+    } = this.props;
     const config = ConfigStore.get('config');
     const loggingStrategy = findNestedPropertyInObject(
       config,
       'uiConfiguration.plugins.mesos.logging-strategy'
     );
 
     if (loggingStrategy === SYSTEM_LOGS) {
-      return <TaskSystemLogsContainer {...props} />;
+      return <TaskSystemLogsContainer task={task} />;
     }
 
-    return <TaskFileViewer limitLogFiles={['stdout', 'stderr']} {...props} />;
+    return (<TaskFileViewer limitLogFiles={['stdout', 'stderr']}
```

Comment:
You also have limitLogFiles twice 🤔 
I've changed it to just be `limitLogFiles={['stdout', 'stderr']}` without taking a prop for it, since that was the original intention